### ISSUE TITLE
CLI: setting grammar and language from command line

### ISSUE THREAD

**Issue by heatherleaf** [MUSTE-Project/MULLE#33]
Creating a new sub-issue to #26, for clarity.

I would like option(s) to set grammar and language from the command line. 
`--language` is already implemented, so `--grammar` is left to do. 

Both options should be compulsory, so no defaulting to Exemplum or Swedish or so. 

Because of that, I would like to have short-options, e.g. `-G` and `-L`.
(I know that you already used `-L` for `--limit-search`, but that's not as important to have as short).

**Comment by fredefox** [comment]
Just an FYI. The options are pretty easy to customize, so if you want to make small tweaks you can do it here:

https://github.com/MUSTE-Project/MULLE/blob/04ecca5b602fe8b11f7db0d13a91b1ec8bef6027/muste/app/Options.hs#L20-L61

For instance to remove `-L` as an alias for `--search-depth` you would just kill line 33.

I'm currently reading your paper draft.

**Comment by heatherleaf** [comment]
thanks, fixed for option `--language` in commit d8332af0c06f065ef80c2f732e6bcf30d2b6c9d2

**Comment by heatherleaf** [comment]
Thanks! 

But, why 3 options? It seems that `-l`/ `--language` is not used now. 

If you didn't think about something special, I suggest:
- remove option `--grammar-lang`
- capitalise the short option for `--language` (to `-L`)
- use `lang` instead of `grammarLang` in line 25 in `muste/app/Main.hs`

**Comment by fredefox** [comment]
They are slightly different. I'm not entirely clear about the semantics, however. Note the example values I provide:

      -G,--grammar GRAMMAR     The grammar to use. E.g. "novo_mod/Exemplum". Please
                               note that this is not actually a path, but rather
                               must be one of the built in grammars.
      -L,--grammar-lang GRAMMAR_LANG
                               The grammar/lang to use. E.g. "ExemplumSwe"
      -l,--language LANG       The language to use. E.g. "Swe"

**Comment by heatherleaf** [comment]
Yes, but where do you use the language "Swe"? The only reference to language in the code (afaik) is "ExemplumSwe", i.e. the concrete Swedish Exemplum grammar.
